src/stylesheets/footer.less

```diff
@@ -0,0 +1,18 @@
+@import 'colors.less';
+
+.footer {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 1rem;
+  color: @link-text;
+  background-color: @footer;
+  text-align: center;
```

Comment:
i don't think you want this re: mockup (https://docs.google.com/document/d/1sPEuJAViYUa5mE7jE0Acen6-mPjbKCzJ2XAVsKEoqmw/edit)